relay/polkadot/src/lib.rs

```diff
@@ -1690,22 +1690,8 @@ pub type Migrations = migrations::Unreleased;
 pub mod migrations {
 	use super::*;
 
-	/// Upgrade Session keys to include BEEFY key.
-	/// When this is removed, should also remove `OldSessionKeys`.
-	pub struct UpgradeSessionKeys;
-	impl frame_support::traits::OnRuntimeUpgrade for UpgradeSessionKeys {
-		fn on_runtime_upgrade() -> Weight {
-			Session::upgrade_keys::<OldSessionKeys, _>(transform_session_keys);
```

Comment:
Also remove `OldSessionKeys` struct and `fn transform_session_keys` as they are unused.

Reply:
thank you, removed.

@ordian I think, lets remove now and then re-introduce when needed.